docker-compose.yml

```diff
@@ -113,7 +113,8 @@ services:
       - DATABASE_USER=insights
       - DATABASE_PASSWORD=insights
       - API_PATH_PREFIX=/api/rbac
-      - DEVELOPMENT=True
+      - DEVELOPMENT=False
+      - BYPASS_BOP_VERIFICATION=True
```

Comment:
What's the purpose of this change? Just curious

Reply:
I'm also curious :thinking:

Reply:
We need this to be able to have a user that is not _Org. Administrator_ in local development. 

The `DEVELOPMENT=False` is to pass an explicit identity. Otherwise, [dev_middleware.py](https://github.com/RedHatInsights/insights-rbac/blob/0200aac03ee2064fdad9f11cf885a8264b12db27/rbac/rbac/dev_middleware.py#L33) could alter it. _(see. [relevant Slack thread](https://ansible.slack.com/archives/CE6Q8FR99/p1652210324932339?thread_ts=1652105183.338989&cid=CE6Q8FR99))_